users/dhertz/dhertz.h

```diff
@@ -0,0 +1,26 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
+
+#undef TAPPING_TERM
```

Comment:
`TAPPING_TERM` should go in your config.h file, NOT here, as it won't be picked up properly. 

Like this: https://github.com/qmk/qmk_firmware/blob/master/users/drashna/config.h